tests/test_planning.py

```diff
@@ -0,0 +1,13 @@
+import pytest
+from planning import *  # noqa
+
+
+romania = GraphProblem('Arad', 'Bucharest', Fig[3, 2])
+
+
+def test_hierarchicalSearch():
```

Comment:
I don't get it -- how is this a hierarchical search? You could use a routing problem as an example if you want, but then you need to define higher level actions (which span several cities, and can be refined to city-to-city substeps).